flepimop/gempyor_pkg/src/gempyor/utils.py

```diff
@@ -137,7 +158,32 @@ def wrapper(*args, **kwargs):
     return decorator
 
 
-def search_and_import_plugins_class(plugin_file_path: str, path_prefix: str, class_name: str, **kwargs):
+def search_and_import_plugins_class(plugin_file_path: str, path_prefix: str, class_name: str, **kwargs: dict[str, any]) -> any:
```

Comment:
The all lowercase `any` refers to the function, I think `typing.Any` is what was intended here?